packages/network-controller/src/NetworkController.ts

```diff
@@ -1055,7 +1059,7 @@ export class NetworkController extends BaseControllerV2<
   ): Promise<string> {
     const sanitizedNetworkConfiguration: NetworkConfiguration = pick(
       networkConfiguration,
-      ['rpcUrl', 'chainId', 'ticker', 'nickname', 'rpcPrefs'],
+      ['rpcUrl', 'chainId', 'ticker', 'nickname', 'rpcPrefs', 'iconUrl'],
```

Comment:
should we do url validation on this like we do for rpcUrl?

Reply:
yup!